app/scripts/modules/core/src/artifact/imageSourceSelector.component.ts

```diff
@@ -12,22 +19,20 @@ const imageSourceSelectorComponent: IComponentOptions = {
         <span ng-bind-html="ctrl.imageSourceText"></span>
       </div>
     </div>
-    <render-if-feature feature="artifacts" ng-if="!ctrl.imageSourceText">
-      <div class="form-group">
-        <div class="col-md-3 sm-label-right">
-          Image Source
-          <help-field key="{{ ctrl.helpFieldKey }}"></help-field>
-        </div>
-        <div class="col-md-7">
-          <div class="radio" ng-repeat="imageSource in ctrl.imageSources">
-            <label>
-              <input type="radio" ng-model="ctrl.command[ctrl.idField]" value="{{ imageSource }}">
-              {{ imageSource | robotToHuman }}
-            </label>
-          </div>
+    <div class="form-group" ng-if="!ctrl.imageSourceText && ctrl.artifactsEnabled">
```

Comment:
Similarly, the only non-indentation diff here is replacement of `render-if-feature` with `ng-if`.